src/components/input/input.tsx

```diff
@@ -360,12 +360,12 @@ export class Input implements LabelableComponent, FormComponent, InteractiveComp
   /**
    * @internal
    */
-  @Event() calciteInputFocus: EventEmitter;
+  @Event() calciteInternalInputFocus: EventEmitter;
 
   /**
    * @internal
    */
-  @Event() calciteInputBlur: EventEmitter;
+  @Event() calciteInternalInputBlur: EventEmitter;
```

Comment:
I'm actually trying to remember why these two events are actually labeled `@internal`.  These seem like public events that consumers should have access to.

Reply:
I'm not sure about the inputblur. That native event should bubble up.

The open/close ones seem like they should be public.

Reply:
Should shadowed `<input>` element events bubble up when there are `calciteInput*` event equivalents?  Seems redundant to allow both, but this is probably why these were labeled `@internal` in the first place because we were relying on the native events to bubble.

Reply:
I don't think there can be equivalents because CustomEvents aren't the same as Keydown, Click, Blur, etc.

I think we should let the native events bubble out.

I don't like how we're passing the native events in some Custom events either. Seems kind of odd to do that.

Reply:
I guess I'm trying to understand the distinction between the native events and the custom events.  What are the custom "Input" and "Change" events really there for if the native ones are already bubbling out and usable?

Reply:
> I'm actually trying to remember why these two events are actually labeled @internal.

I think I remember now.  It is most likely for other Calcite components that consume Calcite Input in their shadow DOM for styling or other purposes.

Reply:
> I don't like how we're passing the native events in some Custom events either. Seems kind of odd to do that.

I guess if we chose to stop bubbling the native events, this would be the way to still expose them to listeners via the custom event.

Reply:
FYI, created a discussion for this here: https://github.com/Esri/calcite-components/discussions/4546

Reply:
> I don't think there can be equivalents because CustomEvents aren't the same as Keydown, Click, Blur, etc.

Should we then ditch redundant custom events like `calciteInputBlur` and `calciteInputFocus` and any other ones that can rely on native bubbled events?